### ISSUE TITLE
Partner is only able to see first 100 lines

### ISSUE THREAD

**Issue by buerge3** [TeamRemote/remote-sublime#14]
The tool works great, except that my partner can only see the first 100 lines of the file we are working on. I am the one hosting the session. Any idea what may be causing this?

**Comment by TerminalFi** [comment]
@buerge3 I don't believe this project is being actively worked on. Checkout https://gitlab.com/TheSecEng/livesession it's early early Alpha but seeks to close this GAP.

I am the author of LiveSession
